cpplint.py

```diff
@@ -867,7 +867,8 @@ def PrintErrorCounts(self):
     for category, count in iteritems(self.errors_by_category):
       sys.stderr.write('Category \'%s\' errors found: %d\n' %
                        (category, count))
-    sys.stderr.write('Total errors found: %d\n' % self.error_count)
+    if self.error_count == 0 and self.verbose_level > 0:
```

Comment:
If there are (many) error messages, I assume the user wants to know how many. If there are no error messages, the user already knows there are 0.